apps/prairielearn/src/sync/course-db.ts

```diff
@@ -455,39 +434,36 @@ export async function loadFullCourse(courseId, courseDir) {
   };
 }
 
-/**
- * @template T
- * @param {any} courseId
- * @param {string} filePath
- * @param {InfoFile<T>} infoFile
- * @param {(line?: string) => void} writeLine
- */
-function writeErrorsAndWarningsForInfoFileIfNeeded(courseId, filePath, infoFile, writeLine) {
-  if (!infofile.hasErrorsOrWarnings(infoFile)) return;
+function writeErrorsAndWarningsForInfoFileIfNeeded<T>(
+  courseId: string,
+  filePath: string,
+  infoFile: InfoFile<T>,
+  writeLine: (line?: string) => void,
+): void {
+  if (!hasErrorsOrWarnings(infoFile)) return;
   // TODO: if https://github.com/drudru/ansi_up/issues/58 is ever resolved,
```

Comment:
@nwalters512 I see this is closed as not planned, do you want to keep this comment here? Maybe add an issue in PL (if one does not exist already)?

Reply:
Done in 91ae79f8e8dd2984812f2d7ec4f238e725177242.